myhpi/static/js/print_processor.js

```diff
@@ -1,67 +1,69 @@
-
 function processPageForPrinting() {
-    moveLinksToFooter();
-    expandAbbreviations();
+  moveLinksToFooter()
+  expandAbbreviations()
```

Comment:
There seems to have been a previous practice which is overwritten with the new prettier config. Any reason for that change (e.g. now removing semicolons where previously they were ubiquitous?)

Reply:
i have no clue about prettier, as noted above I disabled it anyways because it kept messing with djhtml. Maybe @SilvanVerhoeven or @dropforge can shed some light on that

Reply:
I think there is no objective reason for the switch - we just had barely anyone writing JS and I prefer without semicolons (cleaner, why write something you don't need). In February, if not even earlier, I used that style and you asked me to share the JS formatter config, so we added prettier yesterday